corehq/apps/users/bulkupload.py

```diff
@@ -277,7 +277,7 @@ def create_or_update_groups(domain, group_specs, log):
     group_names = set()
     for row in group_specs:
         group_id = row.get('id')
-        group_name = row.get('name')
+        group_name = unicode(row.get('name') or '')
```

Comment:
I don't know enough about the library to say for sure.  Given xlsx string values, it returns strings.  Given xlsx numeric values it returns integers.  My assumption is that that's all we'll have to deal with here, but I suppose it's possible this could end up with something stranger if the user passes a date format or formula or something.  Depends on how the library parses the spreadsheet.  What course of action would you recommend?